src/components/nationalDashboard.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import styles from '../styles/dashboard.module.less';
+import IframeResizer from 'iframe-resizer-react';
+import { Select } from 'antd';
+import { DataSourceType, dataSources } from '../content/NationalDashboard';
+
+const { Option } = Select;
+type ControlsPropsType = {
+  onChange: (dataType: string) => void;
+};
+
+const DashboardControls: React.FC<ControlsPropsType> = (props) => {
+  return (
+    <div className={styles.nationalDashboardControl}>
+      <p className={styles.nationalDashboardControlLabel}>Data types: </p>
+      <Select defaultValue={dataSources[0].id} onChange={props.onChange}>
+        {dataSources.map((dataType) => (
+          <Option key={dataType.id}>{dataType.name}</Option>
+        ))}
+      </Select>
+    </div>
+  );
+};
+
+const NationalDashboard: React.FC = () => {
+  const [dataType, setDataType] = useState(dataSources[0]);
+
+  const handleDataTypeChange = (dataType: string) => {
+    let newDataType: DataSourceType | null = null;
+    for (let i = 0; i < dataSources.length; i++) {
+      if (dataSources[i].id === dataType) {
+        newDataType = dataSources[i];
+        break;
+      }
+    }
```

Comment:
This is fine for now but can you put a to-do here to replace this with a `map`? Also, create an issue on Trello